recipes/newrelic/infrastructure/ohi/sql/ms-sql.yml

```diff
@@ -35,8 +35,13 @@ inputVars:
     prompt: "SQL Server Port (default: 1433)"
     default: 1433
 
-install:
+preInstall:
+  info: |2
+      To capture data from the Microsoft SQL Server integration, you'll first need to meet these prerequisites:
+      - Microsoft SQL Server 2008 R2 SP3 or higher
```

Comment:
I don't think it's quite right, I'll have to get back on that one.
I think SQL Server Express is not supported, and at the minimum it might be 2012

Reply:
Not sure either, taken from our docs: https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/microsoft-sql-server-monitoring-integration

Reply:
I know, and our doc is either incorrect or incomplete. I've reached out to the dev team who hasn't gotten back to me. 
SQL Express certainly did NOT work, even on the more recent version of Windows. Our doc is not mentioning that, but probably should.
Also we only supports the 64 bits install for the windows infra agent, which would be windows 2012. 2008 can be a 32 bits version, which would not work for our install. Also windows 2008 is now EOL, so probably not something to refer to.

Reply:
🤯 ouch, so our docs are wrong on multiple parts. 

At any rate, I've updated everything to simple refer back to the docs now. So the source of truth will remain in the docs and it just needs to be updated there.